### PR TITLE
Link React Storybooks Menu to the Repo

### PR DESCRIPTION
**Pull request by kasunben**

### CODE REVIEW

File: src/client/ui/controls.js [hunk 1]
```diff
@@ -168,10 +168,16 @@ export default class StorybookControls extends React.Component {
...
 
+    const h1AStyle = {
+      textDecoration: 'none',
+    };
+
     return (
       <div style={mainStyle}>
         <div style={h1WrapStyle}>
-          <h3 style={h1Style}>React Storybook</h3>
+          <a style={h1AStyle} href="https://github.com/kadirahq/react-storybook" target="_blank">
```
Comment (arunoda) [inline src/client/ui/controls.js]: Won't this add double padding? Do we need to add `h1AStyle` to this?

**Comment by arunoda** [comment]
CC @thinkholic

File: src/client/ui/controls.js (hunk 1 above)
Comment (mnmtanish) [inline src/client/ui/controls.js]: It won't, right?

Comment (mnmtanish) [inline src/client/ui/controls.js]: Maybe we can rename the variable to something like `linkName`

Comment (thinkholic) [inline src/client/ui/controls.js]: @arunoda `h1AStyle` used to set `textDecoration: 'none'` for link

Comment (thinkholic) [inline src/client/ui/controls.js]: @mnmtanish yes. changed the variable name to `linkStyle`

**Pull request merged**